app/com/linkedin/drelephant/spark/heuristics/ConfigurationUtils.scala

```diff
@@ -0,0 +1,205 @@
+package com.linkedin.drelephant.spark.heuristics
+
+import com.linkedin.drelephant.analysis.SeverityThresholds
+
+object ConfigurationUtils {
```

Comment:
Would be good to change the name to have ConfigurationHeuristicsConstants or something else?

Reply:
Renamed.